titus-supplementary-component/job-activity-history/src/main/java/com/netflix/titus/supplementary/jobactivity/store/JooqConfigurationBean.java

```diff
@@ -14,36 +14,34 @@
  * limitations under the License.
  */
 
-package com.netflix.titus.ext.jooqflyway.jobactivity;
-
-import javax.inject.Inject;
+package com.netflix.titus.supplementary.jobactivity.store;
 
 import com.netflix.archaius.api.annotations.Configuration;
 import com.netflix.titus.common.util.SpringConfigurationUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.env.Environment;
 
-@Configuration(prefix = "titus.ext.jooqflyway")
-public class JooqConfiguration {
+@Configuration(prefix = "titus.ext.supplementary.jobactivity")
+public class JooqConfigurationBean implements JooqConfiguration {
```

Comment:
This is not needed anymore. You can use `Archaius2Ext.newConfiguration` to create Archaius style proxy to Spring config.